code/modules/mob/living/carbon/human/species_types/lesserangel.dm

```diff
@@ -0,0 +1,160 @@
+    
+/datum/species/langel
+	name = "Lesser Angel"
+	id = "langel"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	mutant_bodyparts = list("wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "Angel")
+	use_skintones = 1
+	no_equip = list(SLOT_BACK)
+	blacklisted = 1
```

Comment:
Blacklisted means they can be gotten via magic/mirrors. Giving the drawbacks I think its safe to not blacklist them

Reply:
Thanks for informing me of that, if you got more suggestions and code modification idea, you can contact me directly via email at David.Moussette@gmail.com